pandas/_testing/__init__.py

```diff
@@ -803,23 +803,23 @@ class SubclassedSeries(Series):
 
     @property
     def _constructor(self):
-        return SubclassedSeries
+        return lambda *args, **kwargs: SubclassedSeries(*args, **kwargs)
```

Comment:
can you add a comment to one of these pointing back to the relevant issue

Reply:
Yes, done